### DISCUSSION TITLE
Ability to select the next timer with the cursor if you want to skip

### DISCUSSION THREAD

**Discussion by ghost** [Hanziness/AnotherPomodoro#191] [Ideas & feature requests]
Well, something like that:
- **before**
![before](https://user-images.githubusercontent.com/39465679/159996612-953387c5-e631-4fc4-bef8-3b41212fed46.png)
- **after**
![after](https://user-images.githubusercontent.com/39465679/159996684-21c206a5-eda8-48c3-bc49-69392a10207c.png)

**Comment by Hanziness** [comment]
Cool idea, I just have to make sure that the user won't accidentally move their schedule by clicking on a tile. Maybe a double click would do?

**Reply by dmlls** [reply]
It actually happened to me several times that I single-clicked on it expecting to move that section. I feel like double-click wouldn't be so obvious in terms of discoverability?

What about a dialog asking for confirmation (allowing the user to "not show again")?

**Reply by Hanziness** [reply]
Hmm, I didn't suspect the section tiles to seem "clickable", but I'll think about adding the feature - it doesn't change much if someone accidentally clicks it as they can quickly get back into a state where they were before (the tiles shouldn't be clickable while the timer is running, anyway).
